pkgs/by-name/ma/materialgram/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  tdesktop,
```

Comment:
I remember aliases is not allowed in nixpkgs.

https://github.com/NixOS/nixpkgs/blob/cafba5f9aa857651d6e59ebec7f759e6c56dc107/pkgs/top-level/aliases.nix#L1309